conf.py

```diff
@@ -0,0 +1,52 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
(likely non-actionable for now) The build comes up with many warnings. They're cosmetic so no big deal. Eventually I think we should enforce a build without warnings but it's too early for that.